generative_data_prep/utils/logger_config.ini

```diff
@@ -0,0 +1,30 @@
+[loggers]
```

Comment:
The standard name for this file is `logging.conf`

And this file is generally placed in the root-level, in '<src>/configs' directory or in `<src>` directory.